tyr-runner/src/test/java/org/jboss/tyr/check/DescriptionCheckTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2019 Red Hat, Inc, and individual contributors.
```

Comment:
2021?

Reply:
This class was created back in 2019, originally it was requiredRowsCheck, I added 2 test and rename it to better reflect it's content. Should I change the year anyway?

Reply:
yes, the copyright then should be 2019-2021.